pywbem/_subscription_manager.py

```diff
@@ -1190,14 +1200,27 @@ def _create_destination(self, server_id, dest_url, owned):
                         dest_inst = server.conn.GetInstance(dest_path)
                         self._owned_destinations[server_id][i] = dest_inst
                     return dest_inst
-            dest_path = server.conn.CreateInstance(dest_inst)
+            dest_path = server.conn.CreateInstance(dest_inst,
+                                                   namespace=server.interop_ns)
             dest_inst = server.conn.GetInstance(dest_path)
             self._owned_destinations[server_id].append(dest_inst)
         else:
             # Responsibility to ensure it does not exist yet is with the user
             dest_path = server.conn.CreateInstance(dest_inst)
             dest_inst = server.conn.GetInstance(dest_path)
 
+        # If systemname does not yet exist, create it
+        if server_id not in self._systemnames:
+            self._systemnames[server_id] = dest_inst['SystemName']
```

Comment:
For consistency, use the SystemName key from the path and not from the instance.

Reply:
Whole thing removed since we are depending on the SystemName from the WBEMServer